dev-libs/atk/atk-2.30.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit gnome.org meson multilib-minimal
+
+DESCRIPTION="GTK+ & GNOME Accessibility Toolkit"
+HOMEPAGE="https://wiki.gnome.org/Accessibility"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris ~x86-winnt"
+IUSE="+introspection nls test"
+
+RDEPEND="
+	>=dev-libs/glib-2.34.3:2[${MULTILIB_USEDEP}]
+	introspection? ( >=dev-libs/gobject-introspection-1.32.0:= )
+"
+DEPEND="${RDEPEND}
+	>=dev-lang/perl-5
+	dev-util/glib-utils
+	>=dev-util/gtk-doc-am-1.25
```

Comment:
gtk-doc-am is autotools only; This needs to be `IUSE=gtk-doc`, `gtk-doc? ( >=dev-util/gtk-doc-1.25)` and appropriate `$(meson_use_enable gtk-doc docs)` instead